pydatastructs/linear_data_structures/algorithms.py

```diff
@@ -787,3 +788,35 @@ def longest_common_subsequence(seq1: OneDimensionalArray, seq2: OneDimensionalAr
                     check_mat[i][j] = check_mat[i][j-1]
 
     return OneDimensionalArray(seq1._dtype, check_mat[row][col][-1])
+
+def is_sorted(array, start, end):
```

Comment:
Please rename this function to `is_ordered` and accept an extra parameter `comp`.